src/StoreApi/Routes/V1/CartUpdateCustomer.php

```diff
@@ -44,16 +44,14 @@ public function get_args() {
 						'type'              => 'object',
 						'context'           => [ 'view', 'edit' ],
 						'properties'        => $this->schema->billing_address_schema->get_properties(),
-						'sanitize_callback' => [ $this->schema->billing_address_schema, 'sanitize_callback' ],
-						'validate_callback' => [ $this->schema->billing_address_schema, 'validate_callback' ],
+						'sanitize_callback' => null,
```

Comment:
Removed early sanitization and validation so we can factor in the `customer` object. Otherwise, these functions run too early and we cannot access customer and cart data.